eva-accession-import-automation/create_fasta_from_assembly_report.sh

```diff
@@ -39,11 +39,20 @@ do
     set -o pipefail
     times_wget_failed=0
     max_allowed_attempts=5
+
+    # Check if the contig belong was already downloaded
+    already_downloaded=`grep -c "${genbank_contig}" ${output_folder}/written_contigs.txt`
```

Comment:
Since the contig has a dot in it, we might have to tell grep explicitly to not treat that dot as a regular expression (use the -F switch). I guess this won't be an issue going with the assumption that genbank contigs are of fixed length but we have had bad history with assumptions!